packages/react/dropdown-menu/src/DropdownMenu.tsx

```diff
@@ -18,58 +18,93 @@ type Direction = 'ltr' | 'rtl';
 const DROPDOWN_MENU_NAME = 'DropdownMenu';
 
 type DropdownMenuContextValue = {
```

Comment:
Should we rename this one `DropdownMenuRootContextValue`?

Reply:
I dunno, we didn't do this in `Menu` because usually our context names are like:

- `{ComponentName}Provider` 
- `{ComponentName}ContextValue`

And in this case it's a `DropdownMenuProvider` for `DropdownMenu`. 

We did it this way in `Menu` too but happy to change them both in this PR if you think it makes more sense?

Reply:
Yeah I agree in general because we only have one context to one component ratio.
Here we have one context provider too, but its type can be 2 things.

Reply:
Would you expect me to declare the type alongside the `DropdownRootMenu` component or leave it at the top like this?

Reply:
No I think I'd still group them as they are mostly just related to the context itself.

Reply:
Cool, done 👍